src/js/containers/generateDetachedFiles/DetachedFileAContainer.jsx

```diff
@@ -123,6 +124,6 @@ DetachedFileAContainer.propTypes = propTypes;
 DetachedFileAContainer.defaultProps = defaultProps;
 
 export default connect(
-    (state) => ({ session: state.session }),
+    (state) => ({ session: state.session, agencyList: state.agencyList }),
```

Comment:
`agencyList` is missing in `propTypes`. However, if you're able to store the agency name in DetachedFileA component state, you won't need to access the agency list.